### DISCUSSION TITLE
Add a method to globally stop/resume the internal event loop

### DISCUSSION THREAD

**Discussion by devunt** [Hexagon/croner#256] [Ideas]
Is there a way to stop/resume the internal event loop?
I want to use croner in a distributed manner with some leader-election algorithm enabled, so only the elected leader fires cron jobs.
I need to stop the event loop when the current node is demoted from being leader, and resume it after being promoted.

**Comment by Hexagon** [comment]
There is currently no way to control croner "globally", but each scheduled job can be controlled using `.pause()` and `.resume()`. Will that work for you? You could use the exported variable `scheduledJobs` (array) to enumerate all jobs, if you do not want to keep track of them yourself.

```ts
import { Cron, scheduledJobs } from /*...*/

// ...
```
